src/commands/scale.ts

```diff
@@ -0,0 +1,84 @@
+import { Flags } from '@oclif/core';
+import chalk from 'chalk';
+import inquirer from 'inquirer';
+import Account from '../architect/account/account.entity';
+import AccountUtils from '../architect/account/account.utils';
+import Environment from '../architect/environment/environment.entity';
+import { EnvironmentUtils } from '../architect/environment/environment.utils';
+import BaseCommand from '../base-command';
+import { ComponentVersionSlugUtils, ResourceSlugUtils } from '../dependency-manager/spec/utils/slugs';
+import { ComponentVersion } from './components/versions';
+
+export default class Scale extends BaseCommand {
+  static description = 'Scale a service to a specified number of replicas.';
+
+  static flags = {
+    ...AccountUtils.flags,
+    ...EnvironmentUtils.flags,
+    service: Flags.string({
+      description: 'Name of the service to scale',
+      sensitive: false,
+    }),
+    replicas: Flags.integer({
+      name: 'replicas',
+      description: 'Number of desired service replicas',
+      sensitive: false,
+    }),
+  };
+
+  static args = [{
+    sensitive: false,
+    name: 'component_name',
+    description: 'Name of service to scale, ex. account/component:latest',
+  }];
+
+  async run(): Promise<void> {
+    const { args, flags } = await this.parse(Scale);
+
+    const { component_account_name, component_name, tag, instance_name } = ComponentVersionSlugUtils.parse(args.component_name);
```

Comment:
We should probably allow the user to select from a list of components here if they did not enter one.